scryfall.gemspec

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
Seems like this `scryfal.gemspec` file was created inadvertently.  The file that exists currently is `scryfall-sdk.gemspec`.  It's possible one of the commands I gave you was incorrect and created this new file, but it should be deleted.